VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -74,7 +74,7 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
         provider: { VideoStreamViewController() }
     )
     
-    private var observer: PlayerObserver?
+    private var observer: SystemPlayerObserver?
```

Comment:
Lets rename`SystemPlayerObserver` to more general name now. Old good `PlayerObserver` sounds very common :)